apps/vmq_server/src/vmq_http_config.erl

```diff
@@ -41,3 +41,29 @@ config([HttpModule | Rest], Routes) when is_atom(HttpModule) ->
     end;
 config([], Routes) ->
     Routes.
+
+auth_mode(Req, Module) ->
+    Reg = maps:get(ref, Req),
+    Scheme = binary_to_atom(maps:get(scheme, Req)),
+    Opts = ranch:get_protocol_options(Reg),
+    Global = maps:get(
+        Module,
+        vmq_config:get_env(vmq_server, http_modules_auth, #{}),
+        undefined
+    ),
+    lager:info("fff ~p", [maps:get(opts, Opts)]),
```

Comment:
There's a couple of forgotten development info logs, let's take them out.

Reply:
Taken out